unlock_indicator.c

```diff
@@ -606,18 +608,72 @@ void init_colors_once(void) {
     colorgen_rgb(&tmp_rgb, color, &rgb16);
 }
 
-// init_fonts_once() ?
+te_expr *compile_expression(const char *const from, const char *expression, const te_variable *variables, int var_count);
```

Comment:
These prototypes should be in the .h file, or the functions should be static

Reply:
The same reason as the struct - these are functions that are not public / an interface, they are exclusively used by this file - If you still want them moved I can to that.

Reply:
I mean, I think it should be just fine without the prototype before the function - if you need to forward declare them then the prototypes should be at the top of the .c file, but there shouldn't be a need like this.

see [here](https://github.com/Eyenseo/i3lock-color/blob/91c5a495cc0800548b227a4229edb38515030fcb/unlock_indicator.c#L224) for example - just static <func>, no prototype elsewhere